### ISSUE TITLE
No YAML output for failing tests

### ISSUE THREAD

**Issue by Jameskmonger** [scottcorgan/tap-dot#11]
Hi @scottcorgan,

I have this TAP output:

```
# it gets the correct least significant bit [TestCase: 0]
not ok 7 should be equal
  ---
    operator: equal
    expected: 1111
    actual:   0
  ...
```

This should result in the YAML output being displayed by `tap-dot` ([see the TAP 13 specification](https://testanything.org/tap-version-13-specification.html)).

This expected behaviour can be seen in the screenshots in `README`.

![expected](https://camo.githubusercontent.com/24fe9200faa5a15e454d1b449711de62adfa6177/68747470733a2f2f692e636c6f756475702e636f6d2f3730536d76494c7339492e706e67)

However when I then pipe the output to `tap-dot`, I get the following output:

![actual](http://i.imgur.com/nuIErVu.png)

**Comment by searsaw** [comment]
I'm having this same issue.  The command I'm running is `./node_modules/.bin/tape -r babel-register test/**/*.js | ./node_modules/.bin/tap-dot`.

**Comment by luispablo** [comment]
Having same issue

**Comment by Jameskmonger** [comment]
I've done some investigation into this, and it appears that in `tap.on('assert', function (res) {`, `res.error` is this:

    {
        operator: undefined,
        expected: undefined,
        ctual: undefined,
        at: undefined
    }

**Comment by Jameskmonger** [comment]
I've reported this here https://github.com/scottcorgan/tap-out/issues/23

It is an issue with `tap-out`, not an issue with `tap-dot` specifically.

**Comment by Jameskmonger** [comment]
@searsaw, @luispablo, I have fixed this in #12. @scottcorgan needs to merge it but then it will be fixed :smile:
